### ISSUE TITLE
(feature) Asp.Net Core MVC Support

### ISSUE THREAD

**Issue by Kaffiend** [NathanWalker/angular-seed-advanced#291]
**I'm submitting a ...**  (check one with "x")

[*] feature request

**What is the motivation / use case for changing the behavior?**
I have an ASP.Net Core MVC/Web Api project that uses the parent simple seed to output the _Layout.cshtml. But this was more of a hack job to make it work correctly as a proof of concept. Recently I added the goal of including a downloadable Electron App to encapsulate a section of the project. So I decided to pull my team over to utilize this seed. I'd like to know if anyone else is interested in a branch that can be used with MVC to serve up the project. I'd like to build it out with the existing concepts of re usability and project oriented overrides of the seed config to properly set up the project to output correctly. I'd be happy to maintain a server side seed along with a branch to develop the mvc support of this seed and welcome any help with the project if enough people are interested. If for some reason there is some unforeseen detrimental changes required to make this work while maintaining its existing functionality I could host a separate repo for an MVC version of this seed. Im just curious if anyone else is interested in this and if it can be included in an already fantastic seed.

**Separation of Concerns**
-Server and Web API should be separate project from this seed.
-Seed should be configurable at project level to output correct file types to correct paths in Server project.

**Project Structure**
```
-CombinedProjectDir
--ServerProject
---wwwroot
---Views
----Home
---->Index.cshtml
----Shared
---->_Layout.cshtml
--AngularSeedAdvanced
```

**Comment by Kaffiend** [comment]
Proof of concept dev build only on my personal [fork](https://github.com/Kaffiend/angular-seed-advanced) to output to an mvc project.

**Comment by NathanWalker** [comment]
Since this seed will always maintain as close a mirror of the setup from the parent [angular-seed](https://github.com/mgechev/angular-seed), I wouldn't be able to entertain a change in setup as you suggest here.

However I'll leave this open for others to comment if they'd like to see more of your fork as an official companion to this seed.

I would also recommend submitting a PR to reference your fork as a way forward to this type of integration.

**Comment by Kaffiend** [comment]
@NathanWalker Im finishing up the production outputs right now, then ill update my fork. Basically just added one path var to seed.config for mvc project and a couple tasks for outputting to correct directories. 

Nothing actually changes in the functionality of the seed, just an overridable output path in the project.config with few additional gulp tasks like `build.mvc.dev` that includes the copy task to output in the mvc project, and including a _Layout.cshtml with the index.html that gets all the dep injection. There is probably a way to set this up completely at the project level but im still getting familiar with the code base. At which point a simple integration wiki would suffice.

**Comment by Kaffiend** [comment]
@NathanWalker  I was overlooking some of the concepts of the seed. I've refactored into pure project settings and tasks to integrate with MVC on both dev and prod builds. So i can maintain a mvc configured server on my Repository with the required configuration. And we could just add a simple walk through integrations guide on the wiki if that would be ok. My fork has been updated with what it take to work with MVC.
